podium/datasets/tabular_dataset.py

```diff
@@ -111,7 +112,11 @@ def __init__(
                 format = "custom"
             elif format in {"csv", "tsv"}:
                 delimiter = "," if format == "csv" else "\t"
-                reader = csv.reader(f, delimiter=delimiter, **csv_reader_params)
+                reader = iter(
```

Comment:
why is the `iter` required here?

Reply:
Because we need an iterator. Without it, the `next(reader)` calls would throw an error.